bundles/ranvier-combat/lib/Combat.js

```diff
@@ -91,8 +92,19 @@ class Combat {
    * @param {Character} target
    */
   static makeAttack(attacker, target) {
-    const amount = this.calculateWeaponDamage(attacker);
-    const damage = new Damage({ attribute: 'health', amount, attacker });
+    var amount = this.calculateWeaponDamage(attacker);
+
+	//defines variables associated with critical strikes based on a random percentage chance.
+    const critChance = Math.max(attacker.getMaxAttribute('critical') || 0, 0);
+    const critMultiplier = 1.5;
+    const critical = Random.probability(critChance);
+    
+	const damage = new Damage({ attribute: 'health', amount, attacker });
```

Comment:
Also, you should change the `amount` _before_ instantiating `new Damage()` instead of instantiating it with the pre-critical amount then modifying it after the fact